.github/workflows/PR.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  NODE_VERSION: '12.x'                # set this to the node version to use
+
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ env.NODE_VERSION }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+    - name: npm install, build, and test
+      run: |
+        npm install
+        npm run build --if-present
```

Comment:
Does this line actually deploys the code to Azure? Or just builds it?

Reply:
It just builds it. Deploying could be done. But for now, this seemed like a good first step.

We could add deploying to AWS if we merge to master.